### PR TITLE
fix: fix modal & search position & font issue

### PR DESCRIPTION
**Pull request by jyash97**

### CODE REVIEW

File: src/components/SearchPlugin.js [hunk 1]
```diff
@@ -102,11 +104,28 @@ class App extends Component {
         const { openWithModal } = this.props;
         const isValid = appname && credentials;
         const isOpenWithModal = Boolean(openWithModal);
+        let fontFamilyLink = '';
+        if (
+            theme.typography &&
+            theme.typography.fontFamily &&
+            theme.typography.fontFamily !== 'default'
+        ) {
```
Comment (bietkul) [inline src/components/SearchPlugin.js]: Use get from lodash

File: src/components/Search.js [hunk 2]
```diff
@@ -390,7 +390,12 @@ class Search extends Component {
                 ]}
                 placeholder="Search for products..."
                 iconPosition="right"
-                css={{ marginBottom: 20 }}
+                css={{
+                    marginBottom: 20,
+                    position: 'sticky',
+                    top: '10px',
```
Comment (bietkul) [inline src/components/Search.js]: `top: 10` will also work fine

File: src/components/Search.js [hunk 3]
```diff
@@ -734,12 +739,13 @@ class Search extends Component {
...
-                                                (this.getFontFamily(),
-                                                themeType === 'minimal'
-                                                    ? { padding: 10 }
-                                                    : {})
-                                            }
+                                            css={{
+                                                ...this.getFontFamily(),
+                                                padding:
+                                                    themeType === 'minimal'
+                                                        ? '10px'
```
Comment (bietkul) [inline src/components/Search.js]: Same here, just use `10`

File: src/components/SearchPlugin.js (hunk 1 above)
Comment (jyash97) [reply src/components/SearchPlugin.js]: We currently dont have lodash installed and the app uses conditional check I can create a separate PR to use lodash everywhere.

**Comment by siddharthlatest** [comment]
Sure, that sounds good.

File: src/components/SearchPlugin.js (hunk 1 above)
Comment (siddharthlatest) [reply src/components/SearchPlugin.js]: Sure (y)

**Pull request merged**